pyproject.toml

```diff
@@ -107,3 +106,10 @@ overrides = [
     "virtualenv.*",
   ], ignore_missing_imports = true },
 ]
+
+[tool.docformatter]
+recursive = true
+# this need to be shorter as some docstings are r"""...
+wrap-summaries = 119
+wrap-descriptions = 120
+blank = true
```

Comment:
Why, isn't ruff already doing this?

Reply:
First, it has been disabled, see:
https://github.com/tox-dev/pipdeptree/blob/71c92458e1427a8b4fd0f88746b1e50059c950f6/pyproject.toml#L76
and second from personal experience `docformatter` does better job with line lengths